dashboard/config/scripts/levels/Sample Free Response.level

```diff
@@ -8,8 +8,10 @@
     "title": "Submit a Lesson Plan: Routing and Packets",
     "skip_dialog": true,
```

Comment:
:| some of these being booleans and some of them being strings... does that mean Level#skip_dialog? doesn't work?